go/apps/sequential_send/vumi_app.py

```diff
@@ -152,7 +152,9 @@ def send_scheduled_messages(self, conv):
     def send_message(self, batch_id, to_addr, content, msg_options):
         msg = yield self.send_to(
             to_addr, content, endpoint='default', **msg_options)
-        yield self.vumi_api.mdb.add_outbound_message(msg, batch_ids=[batch_id])
+        # XXX: Do we need to do this?
```

Comment:
Probably not given that we now have middleware that knows how to store messages for conversations. The `batch_id` parameter can likely go too. Perhaps we should open an issue for this for later?